src/box/lua/config/applier/box_cfg.lua

```diff
@@ -496,14 +496,35 @@ local function apply(config)
         --
         -- The RO/RW mode remains unchanged on reload.
         local am_i_bootstrap_leader = false
+        local recently_bootstrapped = false
+
         if is_startup then
             local instance_name = configdata:names().instance_name
+
+            recently_bootstrapped =
+                snapshot.get_path(configdata._iconfig_def) == nil 
+
             am_i_bootstrap_leader =
-                snapshot.get_path(configdata._iconfig_def) == nil and
+                recently_bootstrapped and
                 instance_name == configdata:bootstrap_leader_name()
+
             box_cfg.read_only = not am_i_bootstrap_leader
         end
 
+        if recently_bootstrapped then
+            local renew_interval =
+                instance_config:get(configdata._iconfig_def,
+                                    'failover.renew_interval')
+
+            box.internal.recently_bootstrapped = recently_bootstrapped
```

Comment:
Can we do it in some our namespace? For example, assign it somewhere to `configdata`.

Reply:
Thought about it a bit more: maybe `configdata` is not a nice variant to place it, because a new object is created on each reconfiguration. It would be better to place it into some global/singleton object. Maybe `config` module.

Reply:
Probably temporary added it to the config in `config._recently_bootstrapped`. All of the discussed ideas don't seem good enough.